main/webapp/WEB-INF/jetty-web.xml

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE Configure SYSTEM "https://www.eclipse.org/jetty/configure_10_0.dtd">
+<!-- =============================================================== -->
+<!-- Mixin the GZIP Handler                                          -->
+<!-- This applies the GZIP Handler to the entire server              -->
+<!-- If a GZIP handler is required for an individual context, then   -->
+<!-- use a context XML (see test.xml example in distribution)        -->
+<!-- =============================================================== -->
+<Configure class="org.eclipse.jetty.webapp.WebAppContext">
+    <Call name="insertHandler">
+        <Arg>
+            <New id="GzipHandler" class="org.eclipse.jetty.server.handler.gzip.GzipHandler">
+                <Set name="minGzipSize"><Property name="jetty.gzip.minGzipSize" deprecated="gzip.minGzipSize" default="2048"/></Set>
```

Comment:
Doesn't this have to be slightly smaller than our default XMX setting we ship with? or am I misunderstanding?